recipe/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
     - numpy 1.11.*
     - openblas 0.2.19|0.2.19.*
     - python
-    - r-base
```

Comment:
It should be OK since @johanneskoester fixed it some time ago. You may need to add some annoying extra skips though. See https://github.com/conda-forge/r-gsw-feedstock/commit/c63e0c5a48f0f34492a2ad5b71715c3eff9fdfbb